Source/ACE.Server/Managers/PropertyManager.cs

```diff
@@ -462,7 +462,8 @@ public static void LoadDefaultProperties()
 
         public static readonly ReadOnlyDictionary<string, string> DefaultStringProperties =
             DictOf(
-                ("motd_string", "Welcome to Asheron's Call" + "\n" + "  powered by ACEmulator  " + " \n" + "\n" + "For more information on commands supported by this server, type @acehelp" + "\n")
+                ("welcome_msg", "Welcome to Asheron's Call!\nTo begin your training, speak to the Society Greeter. Walk up to the Society Greeter using the 'W' key, then double-click on her to initiate a conversation."),
+                ("server_motd", "Welcome to Asheron's Call\n  powered by ACEmulator  \n\nFor more information on commands supported by this server, type @acehelp\n")
```

Comment:
i think server_motd should be separate from initial banner text
```
Welcome to Asheron's Call
  powered by ACEmulator  

For more information on commands supported by this server, type @acehelp
```
followed by server_motd if there is one